pilz_control/include/pilz_control/pilz_joint_trajectory_controller_impl.h

```diff
@@ -275,6 +275,16 @@ bool PilzJointTrajectoryController<SegmentImpl, HardwareInterface>::handleMonito
   return true;
 }
 
+template <class SegmentImpl, class HardwareInterface>
+void PilzJointTrajectoryController<SegmentImpl, HardwareInterface>::trajectoryCommandCB(
+    const JointTrajectoryConstPtr& /*msg*/)
+{
+  ROS_WARN_STREAM_NAMED(this->name_,
+                        "For safety reasons the trajectory command interface is deactivated"
+                            << " (for more information see https://github.com/ros-controls/ros_controllers/issues/493)."
```

Comment:
instead of a link to 493 (which I don't find really helpful), I'd suggest a meaningful message (see above).

Reply:
see above

Reply:
See https://github.com/ros-controls/ros_controllers/issues/493 was in fact the reason behind this PR I think it is useful for clarity and transparency to state this. 

Even having two interfaces would be feasible if
1. We ensure only one request is handled at a time
2. There is only one sender/origin for commands

Reply:
Maybe we could split this up into 
Level-WARN: Plain Info for basic user
Level-DEBUG: For more advanced users

Reply:
> Maybe we could split this up into
> Level-WARN: Plain Info for basic user
> Level-DEBUG: For more advanced users

But again:
What is the overall logging concept/idea? What should and what should not be stated?
How do we ensure that the user get's the best possible feedback, in case of errors/problems?

Reply:
https://github.com/PilzDE/pilz_robots/pull/405/commits/c1fa6d7c7dd68bd8eabc3f06d22157eddcd5dd7d Seperated into WARN and INFO